corehq/messaging/scheduling/forms.py

```diff
@@ -185,6 +185,17 @@ class ContentForm(Form):
         required=False,
         label=gettext_lazy("Intervals"),
     )
+    fcm_message_type = ChoiceField(
+        required=False,
+        choices=FCMNotificationContent.MESSAGE_TYPES,
+        initial=FCMNotificationContent.MESSAGE_TYPE_NOTIFICATION,
+        label=''
+    )
+    fcm_action = ChoiceField(
+        required=False,
+        label=gettext_lazy("Action on Notification"),
+        choices=FCMNotificationContent.ACTION_CHOICES,
```

Comment:
nit: do we need an initial here?

Reply:
Probably not required as we have one action as present which is selected by default in the form.
I think once we add more options we can select the preferred action as initial.